soc/xtensa/intel_adsp/tools/cavstool_client.py

```diff
@@ -50,18 +54,51 @@ def send_cmd(self, cmd):
 
     def download(self, filename):
         # Send the FW to server
+        fname = os.path.basename(filename)
+        fsize = os.path.getsize(filename)
+        flen = len(fname)
+
+        md5_tx = hashlib.md5(open(filename,'rb').read()).hexdigest()
+
+        # Pack the header and the expecting packed size is 80 bytes.
+        # The header by convention includes:
+        # size(2), filename(42), MD5(32) and pad(2)
```

Comment:
I'm kinda confused by this math: 2 + 42 + 32 + 2 == 78, not 80. What am I missing?

Reply:
Sorry for my carelessness.  Correct them in both client/sever comments. Thanks!